service/task.go

```diff
@@ -876,6 +876,38 @@ func (uis *UIServer) testLog(w http.ResponseWriter, r *http.Request) {
 				"message":   "problem getting buildlogger logs",
 			}))
 		}
+	} else {
+		// Fallback to legacy test results
+		// Direct link to a log document in the database.
+		testLog, err = model.FindOneTestLog(testName, taskID, taskExec)
```

Comment:
In your case of running this locally, will the test results also live in the Evg DB?

Reply:
yes they will! The testresults query also has a similar fallback.

Reply:
Is it possible then to pass in the `logId` to the rest route so you do not need this third clause in the `if/else` statement?

Reply:
No this change was meant to replicate the same behavior as the cedar route. A user will only have a task id, execution and a test name.

Reply:
Ah ok, makes sense then!